hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,9 @@
+RELEASE_TYPE: minor
+
+This release adds a ``width`` argument to :func:`~hypothesis.strategies.floats`,
+to generate lower-precision floating point numbers for e.g. Numpy arrays.
+
+The generated examples are always instances of Python's native ``float``
+type, which is 64bit, but passing ``width=32`` will ensure that all values
```

Comment:
The release note only mentions `32` bit but we also support `16`

Reply:
And also done.